scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -117,6 +123,62 @@ def _obtain_focused_run(run_number, focus_file_path):
     return focused_ws
 
 
+def _generate_grouped_ts_pdf(focused_ws, q_lims):
+    focused_ws = mantid.ConvertUnits(InputWorkspace=focused_ws, Target="MomentumTransfer", EMode='Elastic')
```

Comment:
Note for second reviewer: This is a rough first cut at getting a workflow together for merging the banks and computing a PDF. In the long term it will become a workflow algorithm but we need to have a scientist test it first so I think leaving this a bit more rough for now is fine.